source/connect/connect-from-the-command-line.txt

```diff
@@ -0,0 +1,133 @@
+.. _comp-command-line-connect:
+
+=============================================
+Connecting from the Command Line with Compass
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+ 
+.. include:: /includes/intro-command-line-connection.rst
+
+There are two ways to start |compass-short| from the command line:
+
+- :ref:`Specify a connection string on the command line
+  <compass-connect-cmd-line>`.
+- :ref:`Specify connection details in a file
+  <compass-connect-file-based>`
+
+If your connection string contains sensitive information, consider
+using a configuration file to avoid exposing that information on
+the command line.
+
+.. _compass-executable-location:
+
+Compass Executable Location
+---------------------------
+
+The name and location of the |compass-short| executable varies by
+operating system. 
+
+Linux
+~~~~~
+The executable is called ``mongodb-compass``. The installer installs it
+in the ``/usr/bin`` directory. 
+
+Windows
+~~~~~~~
+
+The executable is called ``MongoDBCompass.exe``. The installer installs
+it in a folder you pick during the installation process.
+
+MacOS
+~~~~~
+The executable is called ``Compass``. The installer installs
+it under the Applications folder: 
+
+.. code-block:: shell
+
+   /Applications/MongoDB\ Compass.app/Contents/MacOS/MongoDB\ Compass
+
+.. _compass-connect-cmd-line:
+
+Command Line Connection Specification
+-------------------------------------
+
+The command line invocation for |compass-short| has two components, the 
+path to the |compass-short| executable and a connection string. The
+format is:
+
+.. code-block:: shell
+   :copyable: false
+ 
+    <path/to/compass/executable> <connection string>
+
+This example is like the basic connection string for a `MongoDB
+University <https://learn.mongodb.com/>`__ training cluster. Modify the
+connection details to connect to your MongoDB installation:  
+
+.. code-block:: shell
+   :copyable: false
+ 
+   mongodb-compass mongodb+srv://cluster0.cpxklfq.mongodb.net/library
+ 
+
+.. _compass-connect-file-based:
+
+Configuration File Connection Specification
+-------------------------------------------
+
+The command line invocation for |compass-short| can include a reference
```

Comment:
[optional suggestion]

Consider "...can specify a configuration file". I think this is slightly more specific and is consistent with the header.